### DISCUSSION TITLE
fractal dimensionality

### DISCUSSION THREAD

**Discussion by kmytsay** [JeschkeLab/DeerLab#377] [Q&A] [answered]
Is there a way to set fdim as a constant in  bg_homfractal model? Is it reasonable for getting conc = 0 as a result (the data is of low spin concentration)?

**Comment by luisfabib** [comment, marked as answer]
Yes, you can do that by using the `freeze()` method of the corresponding parameter. 

For example, if you have constructed a dipolar model called `Vmodel` which contains the `bg_homfractal` as the background model you can fix the value as follows:
```python
fdim_fixed = 2
Vmodel.fdim.freeze(fdim)
```
and then just run the fit as usual. 

Due to the mathematical nature of the `bg_homfractal` model, it can be that the fractal concentration parameter adopts significantly low values. It might be getting rounded when printing the value. You can check whether the value stored in the fit results object is not exactly zero.

**Reply by kmytsay** [reply]
Awesome, thank you!
